src/components/ProfileMenu/index.tsx

```diff
@@ -37,7 +37,7 @@ const ProfileMenu = ({ activeLink }: ProfileMenuProps) => (
 
     <Link href="/logout" passHref>
       <S.Link>
-        <ExitToApp size={24} />
+        <ExitToApp size={24} title="Sign out"/>
```

Comment:
You forgot a space before closing the tag
```suggestion
        <ExitToApp size={24} title="Sign out" />
```